mapsims/SO_Noise_Calculator_Public_20180822.py

```diff
@@ -0,0 +1,373 @@
+from __future__ import print_function
+import numpy as np
+
+####################################################################
+####################################################################
+### LAT CALCULATOR ###
+####################################################################
+####################################################################
+def Simons_Observatory_V3_LA_bands():
+    ## returns the band centers in GHz for a CMB spectrum
+    ## if your studies require color corrections ask and we can estimate these for you
+    return(np.array([27.,39.,93.,145.,225.,280.]))
+
+def Simons_Observatory_V3_LA_beams():
+    ## returns the LAT beams in arcminutes
+    beam_LAT_27  = 7.4
+    beam_LAT_39  = 5.1
+    beam_LAT_93  = 2.2
+    beam_LAT_145 = 1.4
+    beam_LAT_225 = 1.0
+    beam_LAT_280 = 0.9
+    return(np.array([beam_LAT_27,beam_LAT_39,beam_LAT_93,beam_LAT_145,beam_LAT_225,beam_LAT_280]))
+
+def Simons_Observatory_V3_LA_noise(sensitivity_mode,f_sky,ell_max,delta_ell,N_LF=1.,N_MF=4.,N_UHF=2., apply_beam_correction=True, apply_kludge_correction=True):
+    ## returns noise curves in both temperature and polarization, including the impact of the beam, for the SO large aperture telescope
+    # sensitivity_mode:
+    #     1: baseline, 
+    #     2: goal
+    # f_sky:  number from 0-1
+    # ell_max: the maximum value of ell used in the computation of N(ell)
+    # delta_ell: the step size for computing N_ell
```

Comment:
I assume you don't wanna document anything in this file because users will mostly access `noise.py`, right?